### DISCUSSION TITLE
Unable to update v.1.5.0 to v.1.6.1

### DISCUSSION THREAD

**Discussion by Midnite630** [Luligu/matterbridge#162] [General]
It goes through the motions but never updates. I even uninstalled and reinstalled the HA Add-on.

**Comment by Luligu** [comment]
Hi,
what addon are you using?

**Reply by Midnite630** [reply]
Matterbridge Current version: 2.9.0-PSA-121 with matterbridge-home-assistant

**Reply by Midnite630** [reply]
Matterbridge - Updating matterbridge...
info - [03:50:55.514] [Matterbridge]: Updating matterbridge...
spawn - [03:51:21.352] [Matterbridge:spawn]: added 16 packages, removed 9 packages, and changed 154 packages in 26s
spawn - [03:51:21.352] [Matterbridge:spawn]: 39 packages are looking for funding run `npm fund` for details
spawn - [03:51:21.367] [Matterbridge:spawn]: child process exited with code 0 and signal null
spawn - [03:51:21.368] [Matterbridge:spawn]: child process closed with code 0 and signal null
notice - [03:51:21.368] [Matterbridge]: matterbridge installed correctly
info - [03:51:21.368] [Matterbridge]: Matterbridge has been updated. Full restart required.
info - [03:51:21.369] [Matterbridge]: updating...
info - [03:51:21.371] [PluginManager]: Shutting down plugin matterbridge-home-assistant: Matterbridge is closing: updating......
notice - [03:51:21.373] [PluginManager]: Shutdown of plugin matterbridge-home-assistant completed

**Comment by Luligu** [comment]
Hi, 
kindly check the first rows of matterbridge readme…
